helm/dagster/charts/dagster-user-deployments/templates/deployment-user.yaml

```diff
@@ -111,7 +111,7 @@ spec:
         {{- if $deployment.volumeMounts }}
           volumeMounts:
           {{- range $volumeMount := $deployment.volumeMounts }}
-            - {{ $volumeMount | toYaml | nindent 16 }}
+            - {{- $volumeMount | toYaml | nindent 14 }}
```

Comment:
were these broken? or did the extra spaces just do nothing

Reply:
it was working but looked bad